settings.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd"
+          xmlns="http://maven.apache.org/SETTINGS/1.1.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <servers>
+        <server>
+            <id>bintray</id>
+            <username>${env.BINTRAY_USER}</username>
+            <password>${env.BINTRAY_KEY}</password>
+        </server>
+    </servers>
+    <profiles>
+        <profile>
+            <id>oss-jfrog-snapshots</id>
+            <repositories>
```

Comment:
I mean, if this is needed to build OSIAM, then it does not belong here, but only in the POMs, does it?